src/utility/sdk.js

```diff
@@ -32,12 +32,27 @@ const parseServiceMetadata = (response) => {
 };
 
 const fetchServiceMetadata = async (org_id, service_id) => {
+  if(process.env.REACT_APP_SANDBOX) {
+    return {}
+  }
+
   const apiEndpoint = `${APIEndpoints.GET_SERVICE_LIST.endpoint}/org/${org_id}/service/${service_id}/group`;
   return fetch(apiEndpoint).then(res => res.json()).then(parseServiceMetadata);
 };
 
+const generateOptions = (username) => {
+  if(process.env.REACT_APP_SANDBOX) {
+    return {
+      endpoint: process.env.REACT_APP_SANDBOX_SERVICE_ENDPOINT,
```

Comment:
It is renamed to REACT_APP_CONTRACT_ENDPOINT
PFB the latest .env.example
REACT_APP_CONTRACT_ENDPOINT=
REACT_APP_USER_ENDPOINT=
REACT_APP_SERVICE_PROTOBUF_ENDPOINT=
REACT_APP_SERVICE_EXECUTION_ENDPOINT=
REACT_APP_AWS_IDENTITY_POOL_ID=
REACT_APP_AWS_REGION=
REACT_APP_AWS_USER_POOL_ID=
REACT_APP_AWS_USER_POOL_WEB_CLIENT_ID=
REACT_APP_GOOGLE_CLIENT_ID=

Reply:
What is `REACT_APP_CONTRACT_ENDPOINT`?. This env variable is for local sandbox development.

Reply:
contract API Gateway consists of all the endpoints related to service and org.